App/UI/GreenCertificate/ModalDetail/GreenCertificateTestDetail/GreenCertificateTestDetailView.swift

```diff
@@ -17,632 +17,672 @@ import Models
-        
-        Self.Style.label(diseaseTestLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.disease)
-        Self.Style.label(typeOfTestLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.typeOfTest)
-        Self.Style.label(testResultLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.testResult)
-        Self.Style.label(ratTestNameAndManufacturerLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.ratTestNameAndManufacturer)
-        Self.Style.label(dateTimeOfSampleCollectionLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.dateTimeOfSampleCollection)
-        Self.Style.label(testingCentreLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.testingCentre)
-        Self.Style.label(countryOfTestLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.countryOfTest)
-        Self.Style.label(certificateIssuerLabel, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.certificateIssuer)
-        
-        Self.Style.label(diseaseTestLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.diseaseEn)
-        Self.Style.label(typeOfTestLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.typeOfTestEn)
-        Self.Style.label(testResultLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.testResultEn)
-        Self.Style.label(ratTestNameAndManufacturerLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.ratTestNameAndManufacturerEn)
-        
-        Self.Style.label(dateTimeOfSampleCollectionLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.dateTimeOfSampleCollectionEn)
-        Self.Style.label(testingCentreLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.testingCentreEn)
-        Self.Style.label(countryOfTestLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.countryOfTestEn)
-        Self.Style.label(certificateIssuerLabelEn, text: L10n.HomeView.GreenCertificate.Detail.Label.Test.certificateIssuerEn)
-    
-        Self.Style.closeButton(self.closeButton)
-
+    if let detailTestCertificate = model.greenCertificate.detailTestCertificate {
+      Self.Style.value(self.diseaseTest, text: detailTestCertificate.disease)
+      if let testType = TestType(rawValue: detailTestCertificate.typeOfTest) {
+        Self.Style.value(self.typeOfTest, text: testType.getDescription())
+        Self.Style.value(self.validUntil, text: testType.gedValidUntilValue())
+      } else {
+        Self.Style.value(self.typeOfTest, text: "---")
+      }
+      if let testResultValue = TestResult(rawValue: detailTestCertificate.testResult) {
+        Self.Style.value(self.testResult, text: testResultValue.getDescription())
+      } else {
+        Self.Style.value(self.testResult, text: "---")
+      }
+      if let ratTestNameAndManufacturerValue = detailTestCertificate.ratTestNameAndManufacturer,
+         ratTestNameAndManufacturerValue != ""
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/UI/GreenCertificate/ModalDetail/GreenCertificateTestDetail/GreenCertificateTestDetailView.swift;
  Line: 231;
-->

- :warning: swiftlint (0.39.2): Prefer checking `isEmpty` over comparing `string` to an empty string literal. (empty_string)